Ops/include/OpsWorkerGenetic.h

```diff
@@ -0,0 +1,43 @@
+#ifndef OPSWORKERGENETIC_H
+#define OPSWORKERGENETIC_H
+#include "OpsWorker.h"
+
+#include <random>
+
+namespace Ops {
+
+class WorkerGenetic : public Worker
+{
+public:
+  WorkerGenetic(Evaluator *pEvaluator, MessageHandler *pMessageHandler);
+
+  AlgorithmT getAlgorithm();
+
+  void initialize();
+  void run();
+
+  void setOptVar(const std::string &var, const std::string &value);
+  double getOptVar(const std::string &var, bool &ok);
+
+  void setNumberOfParameters(size_t value);
+
+  void setCrossoverProbability(double value);
+  void setMutationProbability(double value);
+
+private:
+  void selectParents();
+  void crossOver();
+  void mutate();
+  double gaussian(double mean, double dev, double min, double max);
+
+  size_t mParent1,mParent2;
+  std::vector<double> mChild1, mChild2;
+
+  double mCrossoverProbability;
+  double mMutationProbability;
```

Comment:
These two should br initialized with some resonable defaults I think. Or the algorithm should not run if they are not set. (That would actually be better). This is maybe a more general problem to solve..,